DirectProgramming/DPC++FPGA/Tutorials/Features/ac_int/src/ac_int.cpp

```diff
@@ -26,94 +26,76 @@ using MyInt28 = ac_int<28, true>;
 
 void TestBasicOpsInt(queue &q, const int &a, const int &b, int &c, int &d,
                      int &e) {
-  buffer<int, 1> a_buf(&a, 1);
-  buffer<int, 1> b_buf(&b, 1);
   buffer<int, 1> c_buf(&c, 1);
   buffer<int, 1> d_buf(&d, 1);
   buffer<int, 1> e_buf(&e, 1);
 
   q.submit([&](handler &h) {
-    accessor a_acc(a_buf, h, read_only);
-    accessor b_acc(b_buf, h, read_only);
     accessor c_acc(c_buf, h, write_only, no_init);
     accessor d_acc(d_buf, h, write_only, no_init);
     accessor e_acc(e_buf, h, write_only, no_init);
```

Comment:
Do the global stores not take much area?  I guess there's no way for a kernel to return a value other than through an accessor?

Reply:
> Are there any README updates needed with this change?
No, there are no README updates needed. I read through the README but it doesn't go into any detail about how data is transferred to (or from) the device, and it doesn't mention accessors.

Yes, there is no way to return data other than through an accessor or USM pointer.